packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -59,7 +58,11 @@ class DestinationLabel {
   }
 
   /// The icon of the label.
-  final Icon icon;
+  ///
+  /// Typically the icon is an [Icon] widget.  If another type of widget is
```

Comment:
nit: Double space after period should be single space.

Link to `[ImageIcon]`.